prow/jobs/kyma/tools/static-users-generator/static-users-generator.yaml

```diff
@@ -0,0 +1,45 @@
+job_template: &job_template
+  skip_report: true
+  run_if_changed: "^tools/static-users-generator/"
+  decorate: true
+  path_alias: github.com/kyma-project/kyma
+  max_concurrency: 10
+  extra_refs:
+    - org: kyma-project
+      repo: test-infra
+      base_ref: master
+      path_alias: github.com/kyma-project/test-infra
+  spec:
+    containers:
+      - image: eu.gcr.io/kyma-project/prow/test-infra/buildpack-golang:v20181119-afd3fbd
```

Comment:
can't you use the bootstrap image instead?

Reply:
Ok, changed to the bootstrap image